Documentation/GettingStarted.md

```diff
@@ -74,6 +74,23 @@ Pinecil has an unpopulated footprint (U14) for a hall effect sensor (Si7210-B-00
 
 If, after entering sleep mode, the iron still does not see movement for a much longer time (default=10 minutes); it will shut down and return to the home screen.
 
+## Profile Mode (MHP30 only)
+
+On devices that support it, when you long press the button to enter the soldering mode, you enter profile mode which will adjust the temperature according to the profile mode settings.
```

Comment:
Since this may mislead users into thinking that doing this allows adjusting the temperature for profile mode, I think the sentence should be rephrased.
What about:
```suggestion
On devices that support it, a long press on `(+/A)` takes you into profile mode, which initiates the profile selected in the relevant settings.
```